content/how-to/generate-production-build.md

```diff
@@ -1,3 +1,58 @@
 ---
 title: How to Generate a Production Build?
 ---
+
+Generating production builds with Webpack is straight-forward. There are three things to keep in mind:
+
+- Source maps
+- Node environment
+- Minification
+
+## Source maps
+
+We encourage you to have source maps enabled in production. They are useful for debugging and to run benchmark tests. Webpack can generate inline source maps included in the bundles or separated files.
+
+In your configuration, use the `devtools` object to set the source map type. We currently support seven types of source maps. You can find more information about them in our [configuration](https://webpack.github.io/docs/configuration.html#devtool) documentation page.
+
+One of the good options to go is using `cheap-module-source-map` which simplifies the source maps to a single mapping per line.
+
+## Node environment variable
```

Comment:
I think its important to explain in 1-2 sentances _what_ provide plugin does. So:

`blablah creates a FREE compile time global variable. Useful for injecting your node environment as seen below`